docs/contributing/index.md

```diff
@@ -43,6 +43,28 @@ Once you've completed these steps, you are ready to submit your contribution.
 
 Once you've completed these steps, you are ready to submit your contribution.
 
+
+### ...to contribute to docs
```

Comment:
Can we make "proper" headings? I'm not a fan of the ellipsis in headings. I'd like to see the other similar heading updated similarly to the following:

```suggestion
### Contributing to Documentation
```

Reply:
I had this in mind but eventually left it out since all the headings were in the same format. Do you have any specific opinions regarding the other headings?

Reply:
I recommend simply making the same type of change: from "...to X to Y" to "Xing to Y" (e.g., "...to contribute to" becomes "Contributing to")